### PR TITLE
Add parasol and midget RGC

### PR DESCRIPTION
**Pull request by shawntanzk**
fixes https://github.com/obophenotype/cell-ontology/issues/1728 
fixes https://github.com/obophenotype/cell-ontology/issues/1729 
not 100% sure about `'has synaptic IO in region' some 'magnocellular layer of dorsal nucleus of lateral geniculate body'` but the uberon term has the equiv axiom 
```
'layer of lateral geniculate body'
 and ('composed primarily of' some 'magnocellular neurosecretory cell')
```
which I think is right

### CODE REVIEW

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -27632,6 +27634,17 @@ SubClassOf(obo:CL_4023025 ObjectSomeValuesFrom(obo:RO_0002162 obo:NCBITaxon_1009
 SubClassOf(obo:CL_4023025 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0061534))
 SubClassOf(obo:CL_4023025 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_P60041))
 
+# Class: obo:CL_4023026 (midget ganglion cell of retina)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32533915"^^xsd:string) Annotation(oboInOwl:hasDbXref "WikipediaVersioned:Midget_cell&oldid=982299713"^^xsd:string) obo:IAO_0000115 obo:CL_4023026 "A retinal ganglion cell that originate in the ganglion cell layer of the retina, and project to the parvocellular layers of the lateral geniculate nucleus (LGN). These cells are known as midget retinal ganglion cells due to the small sizes of their dendritic trees and cell bodies.")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/contributor> obo:CL_4023026 "https://orcid.org/0000-0001-7258-9596")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/contributor> obo:CL_4023026 "https://orcid.org/0000-0001-7655-4833")
```
Comment (anitacaron) [inline src/ontology/cl-edit.owl:27641]: dce:contributor 😬  check the other term too, tomorrow I'll investigate this QC not failing.

**Review by anitacaron — CHANGES_REQUESTED** [review]

File: src/ontology/cl-edit.owl (hunk 1 above)
Comment (shawntanzk) [reply src/ontology/cl-edit.owl:27641]: omgggg whyyyyyy is my brain like that >.< thankssss you're the best @anitacaron

Comment (shawntanzk) [reply src/ontology/cl-edit.owl:27641]: oh ok i just realised dc:contributor in protege for cl is dce:contributor, that's why its been like that...

Comment (anitacaron) [reply src/ontology/cl-edit.owl]: I'll add the definition and replace the annotation in the 12 cases we have again.

Comment (shawntanzk) [reply src/ontology/cl-edit.owl]: thanks :D why do I have a feeling that the 12 cases are all me >.< hahahaha

**Comment by shawntanzk** [comment]
@emquardokus is this good to go? can I merge this? If so could you give a positive review please? Thanks

**Review by emquardokus — APPROVED** [review]
Looks good, thank you @shawntanzk

**Pull request merged**